main.js

```diff
@@ -17,138 +13,131 @@ function selectItem(e) {
+const inprogressFunctionality = () => {
+  const inprog = document.getElementsByClassName("inprogress");
+  Array.prototype.forEach.call(inprog, node => {
+    node.onclick = function () {
+      const div = this.parentElement;
       console.log(div);
       //nu se poate ajunge aici decat din starea initiala
       if (div.style.backgroundColor != "rgb(136, 136, 136)") {
         div.style.backgroundColor = "yellow";
       }
       // div.classList.add("color-c");     alta modalitate
     };
-  }
+  })
 }
 //butonul finish task, adaugat itemilor existenti
-var myNodelist = document.getElementsByTagName("LI");
-var i;
-for (i = 0; i < myNodelist.length; i++) {
-  var span = document.createElement("SPAN");
-  var txt = document.createTextNode("\u2713");
-  span.className = "finish";
-  span.appendChild(txt);
-  myNodelist[i].appendChild(span);
-}
+Array.prototype.forEach.call(myNodelist, li => {
+  const spanFinish = document.createElement("SPAN");
+  const txtFinish = document.createTextNode("\u2713");
+  spanFinish.className = "finish";
+  spanFinish.appendChild(txtFinish);
+  li.appendChild(spanFinish);
+})
 //functionalitatea butonului finish
-function finishFunctionality() {
-  var finish = document.getElementsByClassName("finish");
-  var i;
-  for (i = 0; i < finish.length; i++) {
-    finish[i].onclick = function () {
-      var div = this.parentElement;
+const finishFunctionality = () => {
+  const finish = document.getElementsByClassName("finish");
+  Array.prototype.forEach.call(finish, node => {
+    node.onclick = function () {
+      const div = this.parentElement;
       //doar daca a fost in progres inainte
       if (div.style.backgroundColor == "yellow") {
         div.style.backgroundColor = "rgb(136, 136, 136)";
         div.classList.add("checked");
       }
     };
-  }
+  })
+}
+function currentNodes(spanx, txt, name, li) {
+  spanx.className = name;
+  spanx.appendChild(txt);
+  li.appendChild(spanx);
 }
-// Nu o folosesc, am pastrat-o ca model pentru toggle
-// var list = document.querySelector("ul");
-// list.addEventListener(
-//   "click",   
-//   function (ev) {
-//     if (ev.target.tagName === "LI") {
-//       // ev.target.classList.toggle("checked");
-//       // ev.target.classList.add("checked");
-//       ev.target.style.backgroundColor= "#888";
-//       ev.target.style.color= "black";
-//       ev.target.style.textDecoration= "line-through";
-//     }
-//   },
-//   false
-// );
-
 // creeaza un nou nod in lista la apasarea butonului add
-function newElement() {
-  var li = document.createElement("li");
-  var inputValue1 = document.getElementById("myInput").value;
-  var inputValue2 = document.getElementById("myInputContent").value;
-  var inputValue = 'Title:  ' + inputValue1 + ' | Content:' + inputValue2;
-  var t = document.createTextNode(inputValue);
+const newElement = () => {
+  const li = document.createElement("li");
+  let inputValue1 = document.getElementById("input-title").value;
```

Comment:
1. Intotdeauna dam nume relevante variabilelor. Codul trebuie sa fie inteles fara nevoia de a adauga comentarii.
2. Incearca sa folosesti un obiect { } in care sa-ti tii cele doua valori.